domains/cdn.poitor.json

```
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "SirOMGitsYOU"
+    },
+    "records": {
+        "A": "188.40.72.254"
```

A records is an Array of strings
```suggestion
        "A": ["188.40.72.254"]
```